app/services/target_service/compatible_target_finder.rb

```diff
@@ -9,7 +9,8 @@ def initialize(target)
     end
 
     def call
-      find_compatible_targets
+      TargetNotification.with(user: @user).deliver_later(targeted_users)
```

Comment:
I think a notification should be sent every time it creates a new conversation. I see that `ConversationService::Creator` receives `targeted_users` but it may not create a conversation for each of them. WDYT?

Reply:
We should also ensure that the conversion is created before actually sending the notification. Looks like we are following the reverse order here

Reply:
@sebastiancaraballo The service discards all the users with conversations created.
Conversations will be created for `targeted_users`. and they will receive notifications.